mne/channels/channels.py

```diff
@@ -403,6 +403,8 @@ def pick_channels(self, ch_names, copy=False):
         -----
         .. versionadded:: 0.9.0
         """
+        if not isinstance(ch_names, (tuple, list, np.ndarray)):
+            ch_names = [ch_names]
```

Comment:
That was the original content of the PR, but when the tests ran, there were a few instances where people were putting in just a single string rather than lists, so the tests broke. That got me thinking that if someone tested it that way, then people are probably intuiting that it's OK to pass a string. So that's why I thought it might be better to allow someone to pass a string rather than force it to be a list.